src/lib/email.js

```diff
@@ -0,0 +1,27 @@
+import sendmail from 'sendmail';
+import StudentStore from '../store/student-store';
+
+const mailer = sendmail({
+    silent: true
+});
+
+export function sendMail(emails){
+    return new Promise((res, rej) => {
+        mailer({
+            from: 'testing@msoe.edu',
+            to: emails.join(','),
+            subject: 'ITEMS DUE SOON',
+            html: 'This is a test.'
```

Comment:
If we want that I feel we should make it another task, as that would take substantially more work.